tests/regionizers/test_administrative_boundary_regionizer.py

```diff
@@ -112,9 +120,46 @@ def test_empty_region_full_bounding_box(toposimplify: Union[bool, float]) -> Non
     assert (
         _merge_disjointed_gdf_geometries(madagascar_result_gdf).difference(madagascar_bbox).is_empty
     )
+    print(madagascar_result_gdf)
     assert "EMPTY" in madagascar_result_gdf.index
```

Comment:
As above, Madagascar probably should be changed since it's replaced by a box.

Reply:
Done ;)